lib/parser/ruby28.y

```diff
@@ -2515,6 +2515,24 @@ keyword_variable: kNIL
                       result = nil
                     }
 
+     f_args_body: f_arglist bodystmt kEND
```

Comment:
@iliabylich Tried to implement it in a straightforward way (mostly by porting [this commit](https://github.com/ruby/ruby/commit/e8f53692ca7d65c92bde6d5bc6a1aea492915d9f) from Ruby). Got stuck with the "4 reduce/reduce conflicts" in the grammar 😞

The problem with `arg` terminal. Not sure what is the proper way to resolve it. Could you please give some hint?

Reply:
a few hints:
1. don't rename rules. I don't see `opt_f_arglist` in https://github.com/ruby/ruby/commit/e8f53692ca7d65c92bde6d5bc6a1aea492915d9f
2. copy all changes related to grammar. partially backported changes may contain conflicts
3. if you are curious and want to dig deeper (which is usually unnecessary) you can pass `--debug -O .racc_output` to `racc` and look into `.racc_output`. It has information about what should be on the stack to get a conflict.

Reply:
Thanks for the advice! It helped 🙂

Made another attempt, and now it works!